<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Prefs_ExternalTools</class>
 <widget class="QWidget" name="Prefs_ExternalTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>External Tools</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>External Tools and Applications</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>732</width>
        <height>969</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>PostScript Interpreter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="psToolLabel">
           <property name="text">
            <string>&amp;Name of Executable:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>psToolLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="psToolLineEdit">
           <property name="toolTip">
            <string>&lt;qt&gt;Add the path for the Ghostscript interpreter. On Windows, please note it is important to note you need to use the program named gswin32c.exe - NOT gswin32.exe. Otherwise, this maybe cause a hang when starting Scribus.&lt;/qt&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="psToolChangeButton">
           <property name="text">
            <string>&amp;Change...</string>
           </property>
           <property name="shortcut">
            <string>Alt+C</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QCheckBox" name="psAntialiasTextCheckBox">
           <property name="toolTip">
            <string>Antialias text for EPS and PDF onscreen rendering</string>
           </property>
           <property name="text">
            <string>Antialias &amp;Text</string>
           </property>
           <property name="shortcut">
            <string>Alt+T</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="psAntialiasGraphicsCheckBox">
           <property name="toolTip">
            <string>Antialias graphics for EPS and PDF onscreen rendering</string>
           </property>
           <property name="text">
            <string>Antialias &amp;Graphics</string>
           </property>
           <property name="shortcut">
            <string>Alt+G</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="psResolutionLabel">
           <property name="text">
            <string>Resolution:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>psResolutionSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="psResolutionSpinBox">
           <property name="suffix">
            <string> dpi</string>
           </property>
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>2400</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Image Processing Tool</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="imageToolLabel">
           <property name="text">
            <string>Name of &amp;Executable:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>imageToolLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="imageToolLineEdit">
           <property name="toolTip">
            <string>&lt;qt&gt;File system location for graphics editor.&lt;/qt&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="imageToolChangeButton">
           <property name="text">
            <string>&amp;Change...</string>
           </property>
           <property name="shortcut">
            <string>Alt+C</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Web Browser</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="webBrowserLabel">
           <property name="text">
            <string>Name of &amp;Executable:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>imageToolLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="webBrowserLineEdit">
           <property name="toolTip">
            <string>&lt;qt&gt;File system location for web browser&lt;/qt&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="webBrowserChangeButton">
           <property name="text">
            <string>&amp;Change...</string>
           </property>
           <property name="shortcut">
            <string>Alt+C</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>PDF Viewer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="pdfViewerLabel">
           <property name="text">
            <string>Name of &amp;Executable:</string>
           </property>
           <property name="buddy">
            <cstring>pdfViewerLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pdfViewerLineEdit">
           <property name="toolTip">
            <string>&lt;qt&gt;File system location for PDF viewer&lt;/qt&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pdfViewerChangeButton">
           <property name="text">
            <string>&amp;Change...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Uniconvertor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="uniconverterLabel">
           <property name="text">
            <string>Name of &amp;Executable:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>imageToolLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="uniconvertorLineEdit">
           <property name="toolTip">
            <string>&lt;qt&gt;File system location for Uniconvertor&lt;/qt&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="uniconvertorChangeButton">
           <property name="text">
            <string>&amp;Change...</string>
           </property>
           <property name="shortcut">
            <string>Alt+C</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Render Frames</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Configurations:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QListWidget" name="latexConfigsListWidget"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QPushButton" name="latexConfigUpButton">
             <property name="text">
              <string>Up</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="latexConfigDownButton">
             <property name="text">
              <string>Down</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="latexConfigAddButton">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="latexConfigDeleteButton">
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="latexConfigChangeCommandButton">
             <property name="text">
              <string>Change...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="latexEditorLabel">
           <property name="text">
            <string>External Editor:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>latexEditorLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="latexEditorLineEdit">
           <property name="toolTip">
            <string>&lt;qt&gt;Path to the editor executable&lt;/qt&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="latexEditorChangeButton">
           <property name="text">
            <string>&amp;Change...</string>
           </property>
           <property name="shortcut">
            <string>Alt+C</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QCheckBox" name="latexEmptyFrameCheckBox">
           <property name="text">
            <string>Start with empty frame</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="latexEmbeddedEditorCheckBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Use Embedded Editor</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QCheckBox" name="latexForceDPICheckBox">
           <property name="toolTip">
            <string>Always use the configured DPI setting for calculating the size, even if the image file reports something different</string>
           </property>
           <property name="text">
            <string>Force DPI</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="latexResolutionLabel">
           <property name="text">
            <string>Resolution:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>latexResolutionSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="latexResolutionSpinBox">
           <property name="suffix">
            <string> dpi</string>
           </property>
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>2400</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>161</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="rescanButton">
       <property name="toolTip">
        <string>Rescan for the external tools if they do not exist in the already specified location</string>
       </property>
       <property name="text">
        <string>&amp;Rescan</string>
       </property>
       <property name="shortcut">
        <string>Alt+R</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>psToolLineEdit</tabstop>
  <tabstop>psToolChangeButton</tabstop>
  <tabstop>psAntialiasTextCheckBox</tabstop>
  <tabstop>psAntialiasGraphicsCheckBox</tabstop>
  <tabstop>psResolutionSpinBox</tabstop>
  <tabstop>imageToolLineEdit</tabstop>
  <tabstop>imageToolChangeButton</tabstop>
  <tabstop>webBrowserLineEdit</tabstop>
  <tabstop>webBrowserChangeButton</tabstop>
  <tabstop>pdfViewerLineEdit</tabstop>
  <tabstop>pdfViewerChangeButton</tabstop>
  <tabstop>uniconvertorLineEdit</tabstop>
  <tabstop>uniconvertorChangeButton</tabstop>
  <tabstop>latexConfigsListWidget</tabstop>
  <tabstop>latexEditorLineEdit</tabstop>
  <tabstop>latexEmptyFrameCheckBox</tabstop>
  <tabstop>latexEmbeddedEditorCheckBox</tabstop>
  <tabstop>latexForceDPICheckBox</tabstop>
  <tabstop>latexResolutionSpinBox</tabstop>
  <tabstop>latexConfigUpButton</tabstop>
  <tabstop>latexConfigDownButton</tabstop>
  <tabstop>latexConfigAddButton</tabstop>
  <tabstop>latexConfigDeleteButton</tabstop>
  <tabstop>latexConfigChangeCommandButton</tabstop>
  <tabstop>latexEditorChangeButton</tabstop>
  <tabstop>rescanButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
